### ISSUE TITLE
Set Youtube quality setting

### ISSUE THREAD

**Issue by gerroon** [andy-portmen/open-in-vlc#34]
Hi

Is it possible to change the quality setting of the Youtube streams? It seems like it is pickign the highest quality and that is pretty choppy in VLC.

thanks

**Comment by andy-portmen** [comment]
Does the newly added FAQ 15 help?
https://add0n.com/open-in-vlc.html?rev

**Comment by gerroon** [comment]
Yeah I saw that, and it was not helping my case, I will give a try again.
